scripts/test.ps1

```diff
@@ -49,6 +49,8 @@ $TFT_Configuration = $Configuration
 $TFT_Pattern = $Pattern
 $TFT_Parallel = $Parallel
 $TFT_All = $All
+$TFB_NetCoreContainers =@("MSTestAdapter.PlatformServices.NetCore.UnitTests.dll")
```

Comment:
nit: would fit better into the env variables on top where all the other patterns are.